content/docs/2.0/scalers/ibm-mq.md

```diff
@@ -0,0 +1,91 @@
++++
+title = "IBM MQ"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on IBM MQ Queue"
+go_file = "ibmmq_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `ibmmq` trigger for IBM MQ Queue.
+
+```yaml
+triggers:
+    - type: ibmmq
+      metadata:
+        host: <ibm-host> # REQUIRED - IBM MQ Queue Manager Admin REST Endpoint
+        queueManager: <queue-manager> # REQUIRED - Your queue manager
```

Comment:
usernameFromEnv and passwordFromEnv is missing

Reply:
Hi @zroubalik, these have been added in the latest commit